src/textarea.js

```diff
@@ -145,6 +145,7 @@ var manageTextarea = (function() {
 
     function popText(callback) {
       var text = textarea.val();
+      text = text.replace(/([ˆ´˜¨]+)/g, '', text);
```

Comment:
Do you need the grouping and what is the third param doing in this context?

`text.replace(/[ˆ´˜¨]/g, '')` does that work?

Reply:
indeed it does, pushed up that revision